### ISSUE TITLE
A set of videos Fermat's Theorem on the sum of 2 squares.

### ISSUE THREAD

**Issue by ahartel** [leios/SoME_Topics#209]
## About the author

I am a math hobbyist with a PhD in Physics and currently working as a software engineer in AI.

## Quick Summary

My idea for this set of videos is to split a proof of **Fermat's Theorem on the sum of 2 squares** up into logical sub-steps.
I want to treat the proof as a graph, where the vertices are theorems (or lemmas if you will) and the edges are implications.
The watcher of the video can then determine by himself how to traverse this graph. He basically chooses his own adventure throught the jungle of mathematics.

The particular proof that I would like to discuss is  the proof that uses Gaussian integers.
The following graph shows a way to split the proof up:

![Screenshot_20220703-164336_Samsung Notes](https://user-images.githubusercontent.com/1397913/177044908-e7641c30-9187-4eb3-9806-5220f577580c.jpg)

Every box in the above graph will probably be an individual video.
The starting video will be the main theorem and the curious watcher of the video can then choose his own adventure by following links at the end of the video.
For example, since the main theorem is an if-and-only-if relation between 2 propeties of numbers, the links at the end of the first video will lead you to the sub-proofs for the implications in both directions.

## Target medium

My target medium is video. I have already prepared a few video scripts.
Since I do not have enough time to become fluent in manim, I decided to produce the visuals by recording my tablet's screen while writing and drawing on it.
I am also having a hard time recording the audio track for my videos.
So I would be happy to find someone who can help me increase video production quality.
I have never made such a video before and it seems horribly hard and seems to need horribly many iterations to match the on-screen content with the voice explaining things.

### More details

to be written

### Contact details

To contact me, please write a comment below this issue.

### Additional context

to be written

**Comment by neondata** [comment]
For those not familiar with the object: [https://www2.informatik.hu-berlin.de/~koessler/Proseminar/Proseminar2012/SchulteSteinberg_zweiquadrate.pdf](url)

**Comment by neondata** [comment]
Choosing a path through math. I would like to contribute to this just to learn about the way of thinking, as physicsts do it differently. I would like to discuss the fact that p = a² + b²  can be written  p - a² - b² = 0 and this can lead to maybe new insight, but nice graphics for sure. Maybe a physicist doing math can better understand a physicist doing math then a mathematician who's thinking is abstract: sm@nascma.com, I'll answer for sure. This will not be playing, but lot of fun!

**Comment by ahartel** [comment]
Great to hear that you're interested in joining. What's your experience with video production and/or math?

**Comment by neondata** [comment]
I quickly made a cmap to show how collaboration could take place and how ideas can be layed out and visualized

https://cmapspublic3.ihmc.us/rid=1Y7F6XW3Z-2DMZV5T-P2/SoME_FermatsTheoremOnTheSumOf2Squares.cmap

**Comment by ahartel** [comment]
Yes, that's unfortunately true, sorry for being unresponsive. I don't think I will find the time to finish this project.

**Comment by neondata** [comment]
Yes, many good ideas suffer from realization. The back side is, that many bad ideas come into existance ;-)
